.env.example

```
@@ -0,0 +1,65 @@
+# Discord Token useful for fetching data; Guild, User, Roles, etc.
+NUXT_OAUTH_DISCORD_BOT_TOKEN=
+
+# Session encryption password (generate a secure random string)
+# You can use: openssl rand -base64 32
+NUXT_SESSION_PASSWORD=
+
+# Useful if you want change to use Cloudflare as Provider
+NUXT_NITRO_PRESET=node-server
+
+# API base url:
+NUXT_PUBLIC_APP_API_BASE_URL="http://localhost:8282"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)